docs/docs/reference/faq.md

```diff
@@ -481,7 +481,7 @@ configs mean, please have a look at the config section.
 
 #### Does it support DELETE calls?
 
-At the moment, the HTTP Sink supports only PUT, POST and PATCH methods.
+Yes, the HTTP Sink now supports DELETE calls also.
```

Comment:
i don't think we need to mention `now`.  we can remove `now` or mention it supports delete call from firehose version x.x.x

Reply:
also since this is a new feature, we need to bump up the firehose version

Reply:
fixed it mas